src/commands/install-lw-scanner.yml

```diff
@@ -0,0 +1,7 @@
+description: >
+  Install the Lacework lw-scanner
+steps:
+  - run:
+      name: Installing the lacework lw-scanner
+      command: |
+        curl https://raw.githubusercontent.com/ipcrm/circleci-orb-lacework/ally-713/src/scripts/install_lwscanner.sh | sudo bash
```

Comment:
can we merge this first into this repository, @ipcrm ?

Reply:
@aedifex added this